discourse.Dockerfile

```diff
@@ -68,9 +58,12 @@ RUN ln -s /usr/share/zoneinfo/UTC /etc/localtime \
 # Run build process.
     && git -C "${CONTAINER_APP_ROOT}" clone --depth 1 --branch "${CONTAINER_APP_VERSION}" https://github.com/discourse/discourse.git app
 
-# Apply patch for LP#1903695
+# Apply patches
```

Comment:
Can we retain the reference to LP#1903695 here? I realise it's in the name of the patch too, but if we're listing references it'd be worth listing them all I think